src/solvers/applyexp.jl

```diff
@@ -0,0 +1,103 @@
+using Printf: @printf
+import ConstructionBase: setproperties
+
+@kwdef mutable struct ApplyExpProblem{State}
+  state::State
+  operator
+  current_time::Number = 0.0
+end
+
+ITensorNetworks.state(A::ApplyExpProblem) = A.state
```

Comment:
```suggestion
state(A::ApplyExpProblem) = A.state
```
since we are in the `ITensorNetworks` module.

Reply:
I'll look into what error I was getting when I didn't prepend `ITensorNetworks`. I agree it shouldn't be needed in principle.